securitycenter/snippets_v2/mute_findings_v2_test.py

```diff
@@ -59,21 +60,15 @@ def test_set_mute_finding(finding):
 def test_set_unmute_finding(finding):
     finding_path = finding.get("finding1")
     response = mute_findings_v2.set_unmute_finding(finding_path)
-
+    assert response.mute.name == "UNMUTED"
 
 
 @backoff.on_exception(
     backoff.expo, (InternalServerError, ServiceUnavailable, NotFound), max_tries=3
 )
 def test_bulk_mute_findings(finding):
     # Mute findings that belong to this project.
-    mute_findings_v2.bulk_mute_findings(
+    response = mute_findings_v2.bulk_mute_findings(
         f"organizations/{ORGANIZATION_ID}","global", f'resource.project_display_name="{ORGANIZATION_ID}"'
     )
-
-    # Get all findings in the source to check if they are muted.
-    response = snippets_findings_v2.list_all_findings(
-        ORGANIZATION_ID,finding.get('source').split("/")[-1],"global"
-    )
-    count, response = enumerate(response)
-    assert len(response)>0
+    assert response.done
```

Comment:
Not sure what this assert statement is verifying. Can this be updated to verify the findings are muted?

Reply:
As this is a long running operation, We will not get response immediately and here we are checking the operation is successfull or not.